app/features/industry/dry.feature

```diff
@@ -0,0 +1,13 @@
+@mvp
+Feature: DRY (don't repeat yourself)
+
+  As an industry user
+  I want data I have already reported be re-used wherever possible
+  So that I do not have to repeat myself
+
+  Scenario: industry user is prompted to import data from a specific SWRS Emissions report (by date)
+  Scenario: admin data from SWRS is re-imported for every filing
+  Scenario: emission data from SWRS is re-imported for every filing
+  Scenario: fuel data from SWRS is re-imported for every filing
+  Scenario: electricity and heat data from SWRS is re-imported for every filing
+  Scenario: production data from SWRS is re-imported for every filing
```

Comment:
I have the understanding that this info is from CIIP only?

Reply:
fixed